pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "dbt-semantic-interfaces"
-version = "0.4.1"
+version = "0.4.2dev0"
```

Comment:
Oh wait, this is wrong!

Reply:
🤔 This is the version that @QMalcolm tole me to use!

Reply:
@tlento Courtney and I talked about this. The goal is to do a dev release so that it can be tested in MetricFlow without doing a production patch release. What we do is a `0.4.2.dev0`, and if all goes well we then follow up with a `0.4.2` release. I don't anticipate anything bad in doing this. The `~=` will automatically ignore versions ending in `.dev`, `rc`, `b`, and `a` so long as none of those specifications exist in the version specified on the right hand side of `~=` specification.

Reply:
Oh is this that it should be `0.4.2.dev0` instead of `0.4.2dev0`. That is likely on me. `.` are expected with `dev` versions but not `rc`, `b`, and `a` strangely 🤔

Reply:
I'll be out the rest of the week, so please move forward without me 🙂